snap/snapcraft.yaml

```diff
@@ -126,7 +133,6 @@ parts:
     - libqt5x11extras5
     - libssl1.1
     - libxml2
-    - libvirt0
```

Comment:
I think we need a:
```yaml
    after:
      - libvirt
```

for the `multipass` part, right?

Reply:
No, this is for staging. The `build-packages:` above make it so `libvirt0` is available during the build anyway.

Reply:
Oh, duh, right :+1: